MetaMask/metamask-mobile
# feat: adding search to the network bottomsheet as part of the networks management UI redesign

## **Description**
Adding search to the network bottomsheet as part of the networks management UI redesign. Here is a 2 minute overview demo: https://www.loom.com/share/dc29313a5cd34e9893c76a0caeba0fc1

Here is a link to the related Figma that includes the adding of the search box to the bottom sheet:
https://www.figma.com/design/76R5BvAVubUhWaN2Ld7MAv/Default-Networks?node-id=1303-9439&t=yhxfHHCsG0CimP0w-0



## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2495

## **Manual testing steps**

1. Add a the following env var that will be used as a feature flag to enable to the feature export MM_NETWORK_UI_REDESIGN_ENABLED="1"
2. From the header in the home screen, click the network selector
3. The network bottom sheet should come up and you should see the search box
4. Type ava in the search box and it should filter out all other networks and show only avalanche
5. For a demo please see the video in the description above

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
